radar-upload-backend/src/main/java/org/radarbase/upload/api/RecordMapperImpl.kt

```diff
@@ -73,12 +74,16 @@ class RecordMapperImpl : RecordMapper {
             limit = limit,
             records = records.map(::fromRecord))
 
-    override fun fromContent(content: RecordContent) = ContentsDTO(
-            url = "${uri.baseUri}/records/${content.record.id}/contents/${content.fileName}",
-            contentType = content.contentType,
-            createdDate = content.createdDate,
-            size = content.size,
-            fileName = content.fileName)
+    override fun fromContent(content: RecordContent): ContentsDTO {
+        val cleanedFileName = URLEncoder.encode(content.fileName, "UTF-8")
+                .replace("+", "%20")
```

Comment:
Why do we replace only this character after encoding? :thinking:

Reply:
It's based on this answer: https://stackoverflow.com/a/31722987/574082. Apparently the encoder works fine, except that it translates spaces to +.

Reply:
Added a test to ensure that the URL is correct with symbols.

Reply:
ok. Thanks for the info